site/server/views/ChartPage.tsx

```diff
@@ -109,11 +109,9 @@ export const ChartPage = (props: {
                         <p>Interactive visualization requires JavaScript</p>
                     </noscript>
                 </main>
+                <SiteFooter />
                 <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=es6,fetch" />
```

Comment:
I've always wondered if these polyfills are necessary, but never looked into it. 

Given that we use Mobx 5 which requires ES6 Proxy support (https://mobx.js.org/README.html#browser-support) I think we can probably remove the polyfills – if the browser supports proxies, I feel it's quite likely it supports the other ES6 features. But then when I load this URL in my browser I still get some polyfills, so maybe that's not quite right.

Anyway, I think we either:
1. Remove the polyfills
2. Move the polyfills to `<SiteFooter>` and load them before anything else (otherwise it doesn't make sense to have them)

What do you think @breck7?

Reply:
Yes I agree let's remove the polyfills script.

But first to be safe let me look into analytics to see what % of our visitors don't have browsers that support ES6.

Reply:
And if we do need to keep ES5 as a target, maybe we could just include the polyfills in our commons.js bundle.

Reply:
> And if we do need to keep ES5 as a target, [...]

Our site is already broken for ES5 browsers because of Mobx – the script fails with an error and pretty much nothing works. To support ES5 we need to downgrade to Mobx 4.

> [...] maybe we could just include the polyfills in our commons.js bundle.

One advantage of the polyfills service is that it checks the user agent and only returns the relevant polyfills

Reply:
> One advantage of the polyfills service is that it checks the user agent and only returns the relevant polyfills

That's smart.

Okay there's no simple way to see exactly who this change might affect. I'll move polyfills to the footer for now and create a separate issue for this so we could roll that out independently.

Reply:
yep makes sense.

Reply:
> But then when I load this URL in my browser I still get some polyfills, so maybe that's not quite right.

@danielgavrilov For what it's worth, there's a new API (v3) available that doesn't return any polyfills for me on chrome/81: https://cdn.polyfill.io/v3/polyfill.js?features=es6,fetch

So it seems like the older API might be delivering too many polyfills?